cf/tasks/kube-dist.sh

```diff
@@ -15,7 +15,7 @@ export PATH=$PATH:$PWD/s3.fissile-binary
 tar -xf s3.stampy-binary/stampy-*.linux-amd64.tgz -C s3.stampy-binary stampy
 export PATH=$PATH:$PWD/s3.stampy-binary
 
-tar -xf s3.certstrap-binary/certstrap-*.linux-amd64.tgz -C s3.certstrap-binary certstrap
+tar -xf s3.certstrap-binary.linux/certstrap-*.linux-amd64.tgz -C s3.certstrap-binary certstrap
```

Comment:
I don't think the output directory exists anymore. (Also, the `PATH` modification on the next line.)